src/components/progress-bar/progress-bar-styles.jsx

```diff
@@ -0,0 +1,170 @@
+import c from 'color';
+
+// This function was created because 'focused' versions on colors were not found in the palette
+const focusColor = color => c(color).darken(0.1).hex();
+const disableColor = color => c(color).lighten(0.7).hex();
+
+const variantModifierFn = (variant, colors) => {
+  const { background, backgroundDisabled, borderColor, color, colorDisabled } = colors;
+
+  const variantDict = {
+    primary: {
+      background,
+      borderColor,
+      borderStyle: 'solid',
+      color,
+
+      '&:hover': {
+        background: focusColor(background),
+        borderColor: focusColor(borderColor),
+      },
+
+      '&:disabled': {
+        background: backgroundDisabled || disableColor(background),
+        borderColor: backgroundDisabled || disableColor(borderColor),
+        color: colorDisabled,
+      },
+
+      '&:focus': {
+        boxShadow: `0 0 0 2px ${focusColor(borderColor)}`,
+      },
+    },
+
+    secondary: {
+      background,
+      borderColor,
+      borderStyle: 'solid',
+      color,
+
+      '&:hover': {
+        borderColor: focusColor(borderColor),
+        color: focusColor(color),
+      },
+
+      '&:disabled': {
+        background: backgroundDisabled || disableColor(background),
+        borderColor: disableColor(borderColor),
+        color: colorDisabled,
+      },
+
+      '&:focus': {
+        boxShadow: `0 0 0 2px ${focusColor(borderColor)}`,
+      },
+    },
+  };
+  return {
+    ...variantDict[variant],
+  };
+};
+
+const sizeModifierFn = ({ typography }, { fontSize, radius }) => ({
+  borderRadius: radius,
+  borderWidth: fontSize < 12 ? 1 : 2,
+  fontSize,
+  fontWeight: fontSize < 12 ? typography.fontWeightLight : typography.fontWeightRegular,
+  height: 2 * radius,
+  width: 2 * radius,
+
+  '& + &': {
+    marginLeft: radius,
+  },
+});
+
+export default theme => {
+  const { palette, transitions, typography, mixins } = theme;
+
+  return {
+    progressBarContainer: {
+      display: 'inline-block',
+    },
+
+    block: {
+      display: 'block',
+      width: '100%',
+    },
+
+    ariaProgressBar: {
```

Comment:
If your intention is to only "show" it for screen readers you might want to look at this:  https://getbootstrap.com/docs/4.0/utilities/screenreaders/ (with an emphasis on _might_)

Reply:
I will have a look at it.